lib/ts/recipe/session/framework/h3.ts

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import Session from "../recipe";
+import type { VerifySessionOptions } from "..";
+import type { SessionEvent } from "../../../framework/h3/framework";
+import { H3Request, H3Response } from "../../../framework/h3/framework";
+import SuperTokens from "../../../supertokens";
+import { eventHandler } from "h3";
```

Comment:
```suggestion
import type { eventHandler } from "h3";
```

Reply:
We are only allowed to import `types` from frameworks cause our package.json has `h3` only in dev dependencies.